ray/operator/manager/manager.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kuberay-operator
+  labels:
+    app.kubernetes.io/name: kuberay
+    app.kubernetes.io/component: kuberay-operator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kuberay
+      app.kubernetes.io/component: kuberay-operator
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kuberay
+        app.kubernetes.io/component: kuberay-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: kuberay-operator
+      containers:
+      - command:
+        - /manager
+#        args:
+#        - --enable-leader-election
+        image: kuberay/operator
```

Comment:
Given these manifests are managed by `opendatahub-io` , should this image be built in `quay.io/opendatahub` ?

Reply:
@VaishnaviHire this is a good question. For other components of ODH, does ODH rebuild images for upstream projects included in ODH? 

In our case, in the `ray/operator/base/kustomization.yaml` we fix the version of the upstream image that we've validated with the rest of the CodeFlare stack. So we are ever only using a working image.  

I'm happy to make any changes you suggest but just for clarity:

1)  How do other components manage their upstream images?
2) Is it possible to mirror the image in our image repository, or are you suggesting rebuilding it as well?

Reply:
We do rebuild images for upstream Projects, however all those components are Tier 0. I think mirroring upstream images should be fine if there are no ODH specific changes.

E.g-
- [Kubeflow Notebook controller](https://github.com/opendatahub-io/odh-manifests/blob/master/odh-notebook-controller/kf-notebook-controller/overlays/openshift/kustomization.yaml#L13)
- [Model Mesh Serving](https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/base/params.env#L5)

Tagging @LaVLaS to confirm if Tier 1 components are required to build/mirror images to `quay.io/opendatahub`

Reply:
I'm happy to mirror the kuberay image over, but https://github.com/opendatahub-io/architecture-decision-records/pull/8 doesn't seem to make any mention of it one way or the other.

Reply:
From offline discussion with @LaVLaS it seems like there is no need to include images in the quay odh org for now. Can we mark this resolved?